config/initializers/clamscan.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+ClamScan.configure do |clam_config|
+  clam_config.client_location = `which clamdscan`.strip
```

Comment:
Is this for the difference between local and dev/stage/prod? May be better as a setting.